src/process/run.cr

```diff
@@ -10,12 +10,13 @@ class Process
   # * `true`: inherit from parent
   # * `IO`: use the given IO
   alias Stdio = Nil | Bool | IO
+  alias Env = Nil | Hash(String, Nil) | Hash(String, String?) |  Hash(String, String)
 
   # Executes a process and waits for it to complete.
   #
   # By default the process is configured without input, output or error.
-  def self.run(cmd : String, args = nil, input = false : Stdio, output = false : Stdio, error = false : Stdio) : Process::Status
-    new(cmd, args, input, output, error).wait
+  def self.run(cmd : String, args = nil, env = nil : Env, unsetenv_others = false : Bool, shell = false : Bool, input = false : Stdio, output = false : Stdio, error = false : Stdio, chdir = nil : String?) : Process::Status
```

Comment:
A better name for `unsetenv_others` would be `inherit_env` and flip the semantic IMO.